docs/guide/url_management.md

```diff
@@ -120,3 +120,32 @@ app.url_checker.handler.custom:
 ```
 
 The `scheme` attribute is mandatory and has to correspond to the name of the protocol, for instance `ftp`.
+
+## URL alias patterns
+
+You can configure how eZ Platform generates URL aliases. The configuration is available under `ezpublish.url_alias.slug_converter_config`, for example:
+
+``` yaml
+ezpublish:
+    url_alias:
+        slug_converter_config:
+            transformation: example_group
+            separator: dash
```

Comment:
we can add available separators somewhere. (dash, underscore and space).
https://github.com/ezsystems/ezpublish-kernel-ee/blob/master/eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php#L373